### PR TITLE
added explanation for Livebook blog post

### PR DESCRIPTION
**Pull request by nishashukla1987**

### CODE REVIEW

**Review by curiosdevcookie — DISMISSED** [review]

File: livebooks/09_29_27_elixir_omt5.livemd [hunk 1]
```diff
@@ -165,6 +165,7 @@ defmodule GoodExample do
     |> maybe_add_min_age(min_age)
     |> maybe_add_max_age(max_age)
     |> Repo.all()
+    |> dbg()
```
Comment (resterle) [inline livebooks/09_29_27_elixir_omt5.livemd]: As nice as this feature is for debugging code you write yourself, i am not sure if it leads to confusion for the user here.

File: livebooks/09_29_27_elixir_omt5.livemd [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+  defp maybe_add_min_age(query, nil), do: query
+  defp maybe_add_min_age(query, min_age), do: where(query, [u], u.age >= ^min_age)
+
+  defp maybe_add_max_age(query, nil), do: query
+  defp maybe_add_max_age(query, max_age), do: where(query, [u], u.age <= ^max_age)
+end
+
+GoodExample.by_age(21, nil) |> Kino.DataTable.new()
+# GoodExample.by_age(nil, 55) |> Kino.DataTable.new()
```
Comment (resterle) [inline livebooks/09_29_27_elixir_omt5.livemd]: Why are these line commented out?

Comment (nishashukla1987) [reply livebooks/09_29_27_elixir_omt5.livemd]: These lines are commented just to call the function with  all possible way and for the users to play around.
What do you think? should we uncomment it or remove it. Its also in the uglyExample.

File: livebooks/09_29_27_elixir_omt5.livemd (hunk 1 above)
Comment (curiosdevcookie) [reply livebooks/09_29_27_elixir_omt5.livemd]: I'd be confused as an Elixir beginner.

**Review by curiosdevcookie — DISMISSED** [review]

**Review by curiosdevcookie — DISMISSED** [review]

**Review by curiosdevcookie — APPROVED** [review]

**Pull request merged**